services/api/config/application.yml.example

```diff
@@ -18,25 +18,35 @@ production:
   secret_token: ~
   blob_signing_key: ~
   sso_app_secret: ~
-  sso_app_id: ~
-  sso_provider_url: ~
-  workbench_address: ~
-  websocket_address: ~
+  sso_app_id: arvados-server
+  sso_provider_url: https://uuid_prefix.your.domain
+  workbench_address: https://workbench.uuid_prefix.your.domain
+  websocket_address: wss://ws.uuid_prefix.your.domain/websocket
   #git_repositories_dir: ~
   #git_internal_dir: ~
+  #git_repo_ssh_base: "git@git.uuid_prefix.your.domain:"
+  #git_repo_http_base: https://git.uuid_prefix.your.domain/
+  #auto_admin_first_user: false
+  #crunch_job_wrapper: :slurm_immediate
+  #sso_insecure: false
```

Comment:
The installation document said the user need to add these lines manually. So either the user don't need to add them (and the cluster can work fine without them), or the user need to add them himself. If the user *does* need to add them manually, why not leave a hint there, ready to be replaced or uncomment?
```auto_admin_first_user``` is an interesting feature and should be noted in the install doc. It can replace the procedure of adding the first user, make it admin and activate it. This saves a lot of time.
```sso_insecure``` should also be noted here and in doc, in case the user is using self-signed certs in test environment.